commands/filescommands/objectcommands/uploaddir.go

```diff
@@ -0,0 +1,235 @@
+package objectcommands
```

Comment:
It's not immediately clear to me why the container would (or should) need to exist for this functionality to work.